knative-operator/pkg/controller/knativekafka/knativekafka_controller_test.go

```diff
@@ -51,7 +51,8 @@ func TestKnativeKafkaReconcile(t *testing.T) {
 		instance: makeCr(withChannelEnabled, withSourceEnabled),
 		exists: []types.NamespacedName{
 			{Name: "kafka-ch-controller", Namespace: "knative-eventing"},
-			{Name: "kafka-controller-manager", Namespace: "knative-eventing"},
+			// TODO: commented out, since rbac injection is missing
```

Comment:
@pierDipi I left them out, on purpose - also I saw your PR to bring this in. Let's land yours after this